command.c

```diff
@@ -1592,36 +1592,24 @@ static void perform_loadhigh(const char *arg)
   __dpmi_int(0x21, &r);
   }
 
-static void perform_loadfix(const char *arg)
-  {
-  const int numblocks = 16;
-  int orig_strat, orig_umblink, allocated, ii;
-  unsigned short allocations[numblocks], allocation, to_64kib, max, size;
+
+#define LOADFIX_NUMBLOCKS 16
+const int loadfix_numblocks = LOADFIX_NUMBLOCKS;
+unsigned short loadfix_allocations[LOADFIX_NUMBLOCKS];
+int loadfix_ii;
+unsigned loadfix_is_v;
+unsigned loadfix_initialised = 0;
+
+static void loadfix_init(unsigned is_v);
```

Comment:
unneeded forward decl?

Reply:
Yeah I think so, during development I imagined I may need to move it further up but that turned out not to be the case.